boltconn/src/app.rs

```diff
@@ -668,13 +678,17 @@ fn parse_two_inbound_service(
     result
 }
 
-fn app_uds_addr() -> &'static str {
+fn app_uds_addr(rootless_mode: bool) -> &'static str {
     #[cfg(target_os = "windows")]
     {
         r"\\.\pipe\boltconn"
     }
     #[cfg(not(target_os = "windows"))]
     {
-        "/var/run/boltconn.sock"
+        if rootless_mode {
+            "/tmp/boltconn.sock"
```

Comment:
I think it would be better to put the socket in `$XDG_RUNTIME_DIR` if it exists and fallback on `/tmp` otherwise.

>  $XDG_RUNTIME_DIR defines the base directory relative to which user-specific non-essential runtime files and other file objects (such as sockets, named pipes, ...) should be stored. The directory MUST be owned by the user, and they MUST be the only one having read and write access to it. Its Unix access mode MUST be 0700. 

https://specifications.freedesktop.org/basedir-spec/latest/